Cython/Includes/cpython/datetime.pxd

```diff
@@ -6,16 +6,16 @@ cdef extern from "Python.h":
 
 cdef extern from "datetime.h":
 
-    ctypedef extern class datetime.date[object PyDateTime_Date]:
+    ctypedef extern class datetime.date[object PyDateTime_Date, check_size ignore]:
```

Comment:
I understand that this is nice for PyPy, but it removes a safety feature in CPython. A warning seems an acceptable price to pay, at least to me.